runelite-api/src/main/java/net/runelite/api/widgets/Widget.java

```diff
@@ -566,14 +538,14 @@
 	void setFontId(int id);
 
 	/**
-	 * Returns the border thickness of the widget
+	 * Returns the border type of item/sprite on the widget
+	 * 0 -> No border
+	 * 1 -> 1px black border
+	 * 2 -> 1px black under 1px white border (selected item)
 	 */
-	int getBorderThickness();
+	int getBorderType();
 
-	/**
-	 * Sets the border thickness of the widget
-	 */
-	void setBorderThickness(int thickness);
+	void setBorderType(int thickness);
```

Comment:
You should at least add `@see` annotation and reference the getter. In generated javadocs they will probably not be close to eachother